### ISSUE TITLE
Please make warning messages (e.g. "Failed to import ...") more verbose

### ISSUE THREAD

**Issue by paulmueller** [ribozz/sphinx-argparse#102]
On several occasions, I ran into problems with sphinx-argparse, because it would give a warning message, e.g. ``WARNING: Failed to import "simple_parser" from "foo.cli"`` without any additional information why the import failed. This is especially cumbersome when trying to build on RTD where a different environment is used (e.g. no tkinter available).

I have set up an example repository here:
https://github.com/paulmueller/RTD-sphinxarg.ext

Running ``sphinx-build . _build/`` only yields the above warning message. The (constructed) problem in this case is the import of tifffile in cli.py: https://github.com/paulmueller/RTD-sphinxarg.ext/blob/master/foo/cli.py#L3

The RTD build is here:
https://readthedocs.org/projects/paulmuellerrtd-sphinxargext/builds/8099029/

It would be nice if the warning would contain more information on the failed import.

Related issues:
- https://github.com/rtfd/readthedocs.org/issues/3032
- https://github.com/rtfd/readthedocs.org/issues/4855

**Comment by dpryan79** [comment]
I can make a simple change that then results in the following:

```
/tmp/RTD-sphinxarg.ext/docs/index.rst:16: WARNING: Failed to import "simple_parser" from "foo.cli".
No module named 'tifffile'
```

Would that suffice?

**Comment by paulmueller** [comment]
Yes

**Comment by dpryan79** [comment]
OK, maybe I can get that pushed out in a new release today still.

**Comment by dpryan79** [comment]
This is now fixed in version 0.2.5, which is available in pypi now and conda probably tonight.
